examples/gno.land/r/gnoland/pages/page_contribute.gno

```diff
@@ -0,0 +1,91 @@
+package gnopages
+
+func init() {
+	path := "contribute"
+	title := "Contributor Ecosystem: Call for Contributions"
+	// XXX: description := "Be a an early adopter and contributor to gno.land through bounties, grants, and Game of Realm's notoable contributions network."
+	body := `
+
+gno.land puts at the center of its identity the contributors that help to create and shape the project into what it is; incentivizing those who contribute the most and help advance its vision. Eventually, contributions will be incentivized directly on-chain; in the meantime, this page serves to illustrate our current off-chain initiatives.
+
+gno.land is still in full-steam development. For now, we're looking for the earliest of adopters; curious to explore a new way to build smart contracts and eager to make an impact. Joining gno.land's development now means you can help to shape the base of its development ecosystem, which will pave the way for the next generation of blockchain programming.
+
+If you are interested in contributing to gno.land, you can jump on in on our [GitHub monorepo](https://github.com/gnolang/gno/blob/master/CONTRIBUTING.md) - where most development happens. Additionally, you can look out to help on specific issues labeled as bounties. All contributions will then concur to form your profile for Game of Realms. In the following sections, we delve into these initiatives in a bit more detail.
```

Comment:
before suggesting bounties, we need to invite them to look at good first issues.

the goal of the contribute page isn't to explain the grants, but to reply to the question I'm always asked: "I'm an <insert-expertise>, how can I contribute?".

we also need to add something about contributing without code.

Reply:
the whole phrase "Additionally, you can look out to help on specific issues labeled as bounties" should be moved in the Gno BOunties section; so that the first paragraph is about the contribution and nothing else.

Reply:
Comment above

Reply:
> we also need to add something about contributing without code.

I think we'll have better pointers to refer people to wrt non-code contributions (ie. gno.me). Can we add this at a later time?

Reply:
I've updated this; lmk wyt.